presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -255,6 +256,15 @@ public SystemSessionProperties(
                         "Maximum amount of distributed total memory a query can use",
                         memoryManagerConfig.getMaxQueryTotalMemory(),
                         true),
+                new PropertyMetadata<>(
```

Comment:
Thanks for raising this PR. Instead of creating a `PropertyMetadata` can we use `SystemSessionProperties#dataSizeProperty`

Reply:
done 
thanks for the review !
+ sent the cla form so this check will eventually pass